sonar-core/src/test/java/org/sonar/core/util/DefaultHttpDownloaderTest.java

```diff
@@ -89,8 +91,7 @@ public void handle(Request req, Response resp) {
           if (req.getPath().getPath().contains("/redirect/")) {
             resp.setCode(303);
             resp.add("Location", "/");
-          }
-          else {
+          } else {
             if (req.getPath().getPath().contains("/timeout/")) {
               try {
                 Thread.sleep(500);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this use of "Thread.sleep()". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2925)